website/docs/usage/visualizers.mdx

```diff
@@ -116,11 +114,9 @@ doc = nlp(text)
 displacy.serve(doc, style="ent")
 ```
 
-import DisplacyEntHtml from 'images/displacy-ent2.html'
-
 <Iframe
   title="displaCy visualizer for entities"
-  html={DisplacyEntHtml}
+  src="/images/displacy-ent2.html"
   height={180}
 />
 
```

Comment:
The table below this line is badly formatted (from before), the last column should be removed from the 2nd and 3rd line. It's currently causing the table not to render properly: https://deploy-preview-12005--spacy.netlify.app/usage/visualizers#ent